navapbc/wic-mt-demo-project-mock-api
# Add wic_agency field.

## Ticket

N/A

## Changes
> What was added, updated, or removed in this PR.

- Added support for a`wic_agency` field to the `/eligibility-screener` endpoint 

## Context for reviewers
> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers. Explain how the changes were verified.

Note that this field is **NOT** included in the CSV export. It would be used for constructing the correct CSV to send to each agency. In other words, a single agency would only see records meant for them, not for other agencies. This functionality is not built out yet.

## Testing
> Screenshots, GIF demos, code examples or output to help show the changes working as expected. ProTip: you can drag and drop or paste images into this textbox.

1. Run `docker-compose up -d --build`
2. Run `make db-upgrade` to run the database migration
3. Navigate to `localhost:8080/v1/docs`
4. Use the swagger UI to create a record. It should return 201 and include the field for the wic agency
5. (Bonus) verify in postgres that the new record has been created including the field for the wic agency
6. 
![Screen Shot 2022-10-27 at 11 17 54 AM](https://user-images.githubusercontent.com/67701/198368737-c03977f4-2c34-4545-be10-ceb90217f6ea.png)
